packages/relay/src/RegistrationManager.ts

```diff
@@ -61,8 +66,8 @@ export class RegistrationManager {
   txStoreManager: TxStoreManager
   logger: LoggerInterface
 
-  lastMinedRegisterTransaction?: EventData
-  lastWorkerAddedTransaction?: EventData
+  lastMinedRegisterTransaction?: CachedEventData
+  lastWorkerAddedTransaction?: CachedEventData
```

Comment:
I don't see where you need the timestamps of these.

Reply:
As we are now transitioning code to work on timestamps, I made all saved `EventData`s to store the block timestamp as well. It is not currently used for some of those. I can return to the original, but I think keeping the timestamp may end up useful.